Lib/test/test_email/test_headerregistry.py

```diff
@@ -873,6 +873,16 @@ def content_disp_as_value(self,
             {'filename': 'foo'},
             [errors.InvalidHeaderDefect]),
 
+        'invalid_value_with_fws_bw_ew': (
```

Comment:
'value' in the preceding test names is the value of the header.  The thing being tested here is the value of the parameter.  Confusing, I know :) Also, I suggest spelling out bw, which at least to my eyes isn't obvious: invalid_parameter_value_with_fws_between_ew".

Also, I suggest adding a test with an EW in the middle of regular tokens, to make sure white space that is *not* between EWs is handled correctly.  Your code should, but we want to guard against future breakage.